packages/collector-status/src/types/collectorstatusTypes.ts

```diff
@@ -0,0 +1,28 @@
+
+import { AlChangeStamp } from '@al/core';
+
+export type StreamType = "offline" | "tmhost" | "syslog" | "tmappliance" | "update" | "updater" | "undefined" | "default";
+export type Status = "ok" | "error" | "warning";
+
+export interface AlCollectorStatusObject {
+    account_id: string;
+    status_id:	string;
+    stream: StreamType;
+    status:	Status;
+    timestamp: string;
+    reported?: AlChangeStamp;
+}
+
+export interface AlCollectorStatusHistoryResponse {
+    rows: number;
+    status_id: string;
+    statuses: Array<AlCollectorStatusObject>;
+}
+
+export interface AlCollectorStatusQueryParams {
+    group_by?: string; // e.g. host_id, stream, source_id, collection_type
```

Comment:
why not create a Group type ?
`export type Group = 'host_id' | 'stream' | 'source_id' | 'collection_type'`